### DISCUSSION TITLE
프로젝트 dependency 추가 제안사항

### DISCUSSION THREAD

**Discussion by Starlight258** [Attica-org/athens-backend#117] [General]
- `spring.dependency-management` : dependency 버전 관리 자동화
의존성 추가할때 버전을 명시하지 않아도 되어서 편해요.
- `actuator` : spring boot 모니터링 도구
모니터링이나 빈 의존관계 등을 보기 편해요.
프로덕션 레벨에서는 필요한 엔드포인트만 선별적으로 노출시켜야합니다. (보안 문제)
- `devtools` : 실시간 재시작
java 클래스 코드가 바뀌면 실시간으로 애플리케이션을 재시작해줘요.
자주 바뀌는 부분(restart)만 올려서 그냥 서버 재시작(cold start)하는 것보다 빨라요.
(프로덕션 레벨에서는 자동 비활성화됩니다.)

**Comment by yybmion** [comment]
이런 dependency가 존재하는 군요 유용해보이네요 👍

**Comment by Starlight258** [comment]
+ `spring configuration processor` : 설정 자동완성 지원

-- + 생각해볼점 : 의존성을 편리함으로 인해 계속 추가하는 것이 좋을까?
